src/components/container/ConnectDevicesMenu/ConnectDevicesMenu.tsx

```diff
@@ -23,34 +24,50 @@ export interface ConnectDevicesMenuProps {
     previewState?: "iOS" | "Android" | "Web" | "ConnectedStates"
     headerVariant?: "large" | "medium"
     connectExternalAccountOptions?: ConnectExternalAccountOptions
+    enableAppleHealthSurvey?: string
```

Comment:
nit rename to appleHealthSurveyName and googleFitSurveyName

Reply:
We may want to keep the `enable...` prefixes on these property names since they intended to specify the "survey to enable X" for each of those sources.  However, I think we have commonly been suffixing with `...SurveyName` for these kinds of things.

An alternative future-proofing approach would be to not assume that the enabling of Apple Health and/or Google Fit will always require a survey, and just let it be something like `onEnableAppleHealth?: () => void`.  Then in the view builder, you can let the user configure an action (which can be to launch a survey).

Reply:
I think we probably want to keep this as SurveyName for now, as the only reliable way to trigger this is by launching a survey - in particular because 1) the survey has to set custom fields or something to add you to the segment and 2) the apps have code which specifically checks for AppleHealth / GoogleFit query rules after surveys have been completed.

Reply:
(I did rename to enableAppleHealthSurveyName, enableGoogleFitSurveyName)